api.swagger.yaml

```diff
@@ -6657,6 +6657,89 @@ paths:
         - SessionCookieHeader: []
         - APIKeyHeader: []
         - CognitoJwt: [ invotra/locations:write ]
+  '/locations/any/{location_id}':
```

Comment:
Maybe we just go with /locations/id on this one?

Reply:
makes sense, done.